distributed/client.py

```diff
@@ -3730,22 +3730,23 @@ def dump_to_file(dask_worker=None):
 
         assert all(len(data) == v for v in response.values())
 
-    def upload_file(self, filename, **kwargs):
```

Comment:
Note that I'm dropping `**kwargs` here as they weren't actually being used anywhere. This isn't strictly necessary for this PR, but seems like a good cleanup thing to do since I'm touching this code anyways.